lib/fathead/django_built_ins/parse.py

```diff
@@ -0,0 +1,177 @@
+from bs4 import BeautifulSoup
+
+DJANGO_HOME = 'https://www.djangoproject.com/'
+DJANGO_DOC_URL = 'https://docs.djangoproject.com/en/1.9/ref/templates/builtins/'
```

Comment:
It would be nice if you can add support for `argparse`, that way you can control of the documentation version you want to generate doing something like:

```bash
python parse.py --doc-version 1.9
```